dashboard/src/components/api/environment/docker-file-parser.spec.ts

```diff
@@ -24,6 +24,36 @@ describe('Simple dockerfile parser >', () => {
     parser = new DockerfileParser();
   });
 
+  describe('parsing comments >', () => {
+
+    it('should parse commented line', () => {
+      const dockerfile = `# commented line
+FROM codenvy/ubuntu_jdk8`;
+
+      const result = parser.parse(dockerfile);
+
+      const expectedResult = [{
+        comment: '# commented line'
+      }, {
+        instruction: 'FROM',
+        argument: 'codenvy/ubuntu_jdk8'
```

Comment:
maybe use "eclipse/ubuntu_jdk8" (I know it's only a test)

Reply:
I have no objections, I will replace all these strings